resume-buddy/src/main/java/com/google/sps/servlets/RevieweeDataServlet.java

```diff
@@ -32,15 +44,50 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     String fname = ServletHelpers.getParameter(request, "fname", "");
     String lname = ServletHelpers.getParameter(request, "lname", "");
     String email = ServletHelpers.getParameter(request, "email", "");
-    reviewee = new Reviewee(fname, lname, email);
+    String resumeURL = getUploadedFileUrl(request, "resume");
+    reviewee = new Reviewee(fname, lname, email, resumeURL);
+
     Entity revieweeEntity = new Entity("Reviewee");
     revieweeEntity.setProperty("first-name", fname);
     revieweeEntity.setProperty("last-name", lname);
     revieweeEntity.setProperty("email", email);
+    revieweeEntity.setProperty("resumeURL", resumeURL);
 
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     datastore.put(revieweeEntity);
 
-    response.sendRedirect("resume-review.html");
+    // response.sendRedirect("resume-review.html");
+    response.sendRedirect("reviewee-form.html");
+  }
+
+  /** Returns a URL that points to the uploaded file, or null if the user didn't upload a file. */
+  private String getUploadedFileUrl(HttpServletRequest request, String formInputElementName) {
+    BlobstoreService blobstoreService = BlobstoreServiceFactory.getBlobstoreService();
+    Map<String, List<BlobKey>> blobs = blobstoreService.getUploads(request);
+    List<BlobKey> blobKeys = blobs.get("resume");
+
+    // User submitted form without selecting a file, so we can't get a URL. (dev server)
+    if (blobKeys == null || blobKeys.isEmpty()) {
+      return null;
+    }
+
+    BlobKey blobKey = blobKeys.get(0);
+
+    // User submitted form without selecting a file, so we can't get a URL. (live server)
+    BlobInfo blobInfo = new BlobInfoFactory().loadBlobInfo(blobKey);
+    if (blobInfo.getSize() == 0) {
+      blobstoreService.delete(blobKey);
+      return null;
+    }
+
+    ImagesService imagesService = ImagesServiceFactory.getImagesService();
+    ServingUrlOptions options = ServingUrlOptions.Builder.withBlobKey(blobKey);
+
+    try {
+      URL url = new URL(imagesService.getServingUrl(options));
+      return url.getPath();
+    } catch (MalformedURLException e) {
+      return imagesService.getServingUrl(options);
+    }
```

Comment:
Since now we are using file upload to upload pdfs, you don't need to use `ImagesService`. 
This documentation explains how to handle serving blob and getting upload url https://cloud.google.com/appengine/docs/standard/java/blobstore